microsoft-edge/extensions-chromium/publish/create-dev-account.md

```diff
@@ -1,39 +1,88 @@
 
-1. Navigate to [account.microsoft.com][MicrosoftAccount].  
-1. Choose **Create a Microsoft account**.
+<!-- ====================================================================== -->
+## Types of accounts related to publishing Microsoft Edge extensions
+
+| Type of account | Description |
+|---|---|
+| _Microsoft account (MSA)_ | An Outlook.com, Live.com, or Hotmail.com account. |
+| _GitHub account_ | A user account at GitHub.com.  You can use your GitHub account's login credentials to login to Partner Center—a Microsoft account (MSA) will be created for you. |
+| _Partner Center account_, _Partner Center developer account_ | A _Partner Center account_ is an account on partner.microsoft.com.  To submit Microsoft Edge extensions, you need a _Partner Center developer account_, which is a Partner Center account that has a Microsoft account (MSA) as the Primary Owner. |
+| _Microsoft Edge Program account_ | Enables multiple users to work with Microsoft Edge extensions at Partner Center. |
+| _Azure Active Directory_, _AD account_, _Azure AD_ | An Azure Active Directory account. |
+| _Azure Active Directory tenant_, _AAD tenant_ | A _tenant_ represents an organization.  A tenant is a dedicated instance of Azure AD that an organization or app developer receives at the beginning of a relationship with Microsoft. |
+
+
+<!-- ====================================================================== -->
+## Determining if you have a Partner Center developer account
+
+1. You need a Microsoft account (MSA).  If you don't have a Microsoft account (MSA), create one.  One way to create a Microsoft account (MSA) is to use your existing GitHub account to log in to Partner Center - dialog boxes help you automatically create a Microsoft account (MSA).
+1. Use that a Microsoft account (MSA) to login to Partner Center.
+1. Register for the Microsoft Edge program, to be able to submit extensions.
+
+To submit an extension to the Microsoft Edge Add-ons website, you must be registered as a developer with the Microsoft Edge program.  The Microsoft Edge program is located on [Partner Center][MicrosoftPartnerCenter].
+
+> [!NOTE]
+> There is no registration fee for submitting extensions to the Microsoft Edge program.
+
+
+<!-- ====================================================================== -->
+## Before you begin
+
+If you don't have a Partner Center account, or you have a Partner Center account but its Primary Owner isn't a Microsoft account (MSA), you must either:
+*  Use an existing Microsoft account (MSA)) to register with the Microsoft Edge program.
+*  Create a new Microsoft account (MSA).
+
+   A Microsoft account (MSA) is an Outlook.com, Live.com, or Hotmail.com account.
```

Comment:
This repetition seems unnecessary since we have defined MSA accounts at the top. (Or atleast, this should be merged in the same sentence as the "Create a new Microsoft account (MSA).